### PR TITLE
added PHP 8 support

### PR DESCRIPTION
**Pull request by oburk**

### CODE REVIEW

**Comment by oburk** [comment]
**ready for review**

File: composer.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
   "license": "MIT",
   "require": {
     "ext-fileinfo": "*",
-    "php": "^5.4|^7.0",
-    "symfony/http-foundation": "^3.1|^4.0|^5.0"
+    "php": "^7.3 || ^8.0",
+    "symfony/http-foundation": "^5.0"
```
Comment (arnegroskurth) [inline composer.json]: Any reason to remove support for the other versions?

File: composer.json [hunk 2]
```diff
@@ -12,11 +12,11 @@
   "license": "MIT",
   "require": {
     "ext-fileinfo": "*",
-    "php": "^5.4|^7.0",
-    "symfony/http-foundation": "^3.1|^4.0|^5.0"
+    "php": "^7.3 || ^8.0",
```
Comment (arnegroskurth) [inline composer.json]: It seems like this library does still support 5.x - any reason to remove that support?

**Review by arnegroskurth — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (oburk) [reply composer.json]: I re added the constrains here

File: composer.json (hunk 2 above)
Comment (oburk) [reply composer.json]: PHPUnit9 requires PHP >= 7.3 and I had problems with running PHPUnit 5. Now I add a new Travis CI job. Please check it again.

**Review by arnegroskurth — APPROVED** [review]

**Pull request merged**